### DISCUSSION TITLE
Assigning colors to the webview for categories..

### DISCUSSION THREAD

**Discussion by jbuszkie** [OpenSageTV/Community#132] [Q&A] [answered]
I was just about to ask here why all my sports don't show up as green anymore.  I found a reply on a different thread about how they removed the "sports" category in guide data.  I also found out you can add your own categories and color them.  SO I created a Baseball, Basketball, Football etc categories and colored them green.  I hope this saves them across all the HD300s!

But along those same lines...  Is there a way to have the Web version use those same custom categories?  Not a big deal..  And maybe I just have to restart sage for the web version to pick them up.  I guess I can try that! lol`

**Comment by jbuszkie** [comment]
UGH!  and it seems like we can't search the old forum either!! The lake of colors for sports is really bugging me on the web interface! lol

**Comment by jbuszkie** [comment, marked as answer]
Ok after a lot of playing around...  instead of doing my day job (don't tell my boss!)
One can update the sage_all.css file to add a couple of things.
I added these lines (in green)
![image](https://github.com/OpenSageTV/Community/assets/11888706/a0c8bd5f-c716-402a-841d-cd1c8cf7aad2)

```
    .epgcell table.show_playoffsports-basketball,
    .epgcell table.show_playoffsports,
    .epgcell table.show_basketball,
    .epgcell table.show_baseball,
    .epgcell table.show_hockey,
    .epgcell table.show_football,
    .epgcell table.show_softball,
    .epgcell table.show_classicsportevent-football,
```


Now this list is not complete and I'll probably have to add more of them as I find sports (that I care about)
after editing the file in 
/sagetv/server/jetty/work/jetty-0_0_0_0-8081-nielm_sagewebserver_war-_nielm_sagewebserver-any-/webapp
(in my installation)  I have to CTRL + refresh to see the changes.
The real thing would be to update the WAR file..  But I'll live with this for now.

Yeah!  My sports are now green again!

Now when I forget what I did I can come back here to remember! 😄

**Comment by wayner9** [comment]
Awesome - I just got around to doing this.  The next thing is to change my custom menus.  I used to have a separate TV submenu for sports but that stopped working a few months ago when this issue popped up.  I think I just need to add all of these category/genre types like basketball, football, etc to the category filter.
